felix/fv/bpf_test.go

```diff
@@ -968,32 +968,32 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 						options.ExtraEnvVars["FELIX_DebugBPFMapRepinEnabled"] = "true"
 					})
 
-					It("should repin maps", func() {
+					It("should not repin maps (option is deprecated)", func() {
```

Comment:
We could completely remove `Describe("with map repinning enabled", func() {` right?